test/kubernetes_unittest.cc

```diff
@@ -23,10 +23,99 @@
 #include "gtest/gtest.h"
 #include "temp_file.h"
 
+#include <boost/network/protocol/http/server.hpp>
 #include <memory>
 
 namespace google {
 
+json::value NodeMetadataWithInstance(const json::value& node) {
+  return json::object({
+    {"blobs", json::object({
+      {"api", json::object({
+        {"version", json::string("1.6")},  // Hard-coded in kubernetes.cc.
+        {"raw", node->Clone()},
+      })},
+      {"association", json::object({
+        {"version", json::string("TestVersion")},
+        {"raw", json::object({
+          {"infrastructureResource", json::object({
+            {"type", json::string("gce_instance")},
+            {"labels", json::object({
+              {"instance_id", json::string("TestID")},
+              {"zone", json::string("TestZone")},
+            })},
+          })},
+        })},
+      })},
+    })},
+  });
+}
+
+json::value PodMetadataWithInstance(const std::string& pod_name,
+                                    const json::value& pod) {
+  return json::object({
+    {"blobs", json::object({
+      {"api", json::object({
+        {"version", json::string("1.6")},
+        {"raw", pod->Clone()},
+      })},
+      {"association", json::object({
+        {"raw", json::object({
+          {"controllers", json::object({
+            {"topLevelControllerName", json::string(pod_name)},
+            {"topLevelControllerType", json::string("Pod")},
+          })},
+          {"infrastructureResource", json::object({
+            {"labels", json::object({
+              {"instance_id", json::string("TestID")},
+              {"zone", json::string("TestZone")},
+            })},
+            {"type", json::string("gce_instance")},
+          })},
+          {"nodeName", json::string("TestSpecNodeName")},
+        })},
+        {"version", json::string("TestVersion")},
```

Comment:
Would it make sense to put the version as the first field, for consistency? Also below.

Reply:
Done.